### PR TITLE
Add unruggable memecoins launch and transfers indexers

### PR DESCRIPTION
**Pull request by Rhumsteak**

### CODE REVIEW

File: infra/indexers/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Fonction pour décoder une chaîne hexadécimale en une chaîne de caractères
```
Comment (fracek) [inline infra/indexers/src/utils.ts:1]: Should the comments be in english? 😂

Comment (fracek) [reply infra/indexers/src/utils.ts:1]: Also I believe starknet.js has a function to convert short strings to/from felts. it's in the `shortString` namespace.

Comment (Rhumsteak) [reply infra/indexers/src/utils.ts]: Sorry about the french comments :D

Ok for the use of the starknet.js function

**Review by fracek — APPROVED** [review]

**Comment by ugur-eren** [comment]
Closing in favour of #250

**Pull request closed without merging**